service/storage_service.cc

```diff
@@ -442,12 +442,18 @@ future<> storage_service::topology_transition(storage_proxy& proxy, cdc::generat
 }
 
 future<> storage_service::merge_topology_snapshot(raft_topology_snapshot snp) {
-   auto s = _db.local().find_schema(db::system_keyspace::NAME, db::system_keyspace::TOPOLOGY);
    std::vector<mutation> muts;
-   muts.reserve(snp.mutations.size());
-   boost::transform(snp.mutations, std::back_inserter(muts), [s] (const canonical_mutation& m) {
-       return m.to_mutation(s);
-   });
+   muts.reserve(snp.topology_mutations.size() + (snp.cdc_generation_mutation ? 1 : 0));
+   {
```

Comment:
Why do you need this local scope?

Reply:
Using variable called `s` (for schema) here and below, wanted to avoid name clashes.